staff/marti-malek/cv/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>CV</title>
+</head>
+
+<body>
+    <header>
+        <h1>Martí Malek</h1>
+        <img src="superrandom" alt="My Image">
+        <h3>Personal Information</h3>
+        <address>
+        Address: C/Pedrell N.80 <b>Barcelona</b><br>
+        Tlf: 697 501 950<br>
```

Comment:
You can add hyperlink to phone number too, similar way to hyperlink the email adress:
```javascript
<a href="tel:8648726348">8648726348</a>
```

Reply:
Thx, did it in the second commit!

Reply:
Thx! I used it on the second commit!